### ISSUE TITLE
RPI error in ofTexture (error: ‘GL_RGB16F’ was not declared in this scope;) 

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#8213]
There is a function being called ofGLESVersionFromGL that tries to parse textual info from glGetString(GL_VERSION); and define which capabilities it has. 
I think there are more direct ways of parsing this info, I'll be testing to just check if GL_RGB16F is defined

- https://github.com/openframeworks/openFrameworks/pull/8212

**Comment by dimitre** [comment]
After some more work there is another apothecary lib with problems, in [rpi-build (armv7l](https://github.com/openframeworks/openFrameworks/issues/8213#logs)
```
  /usr/lib/gcc-cross/arm-linux-gnueabihf/13/../../../../arm-linux-gnueabihf/bin/ld: ../../../libs/kiss/lib/linuxarmv7l/libkiss.a(kiss_fftr.o)(.text+0x34): unresolvable R_ARM_MOVW_ABS_NC relocation against symbol `stderr@@GLIBC_2.4'
```

**Comment by dimitre** [comment]
I've tried to change the formula but now I've find the RPI action is disabled. 
we need help to put everything up to date.

**Comment by danoli3** [comment]
Yeah the problem with package managers! Best to keep it all custom built.

Rpi just needs the cross compiler setup again and we can build for Linux
ripi

**Comment by dimitre** [comment]
no, it is a problem in a custom built

**Comment by dimitre** [comment]
cross compiling is working great in OF actions. it now uses compilers installable via apt get and multistrap
we can copy the same logic to apothecary

**Comment by danoli3** [comment]
Sounds good! I’ll have a look now thanks

**Comment by dimitre** [comment]
closed by #8212
